radicle-cli/examples/rad-patch.md

```diff
@@ -52,6 +52,7 @@ $ rad patch
 Define power requirements 191a14e520f R0 3e674d1 (flux-capacitor-power) ahead 1, behind 0
 └─ * opened by did:key:z6MknSLrJoTcukLrE435hVNQT4JUhbvWLX4kUzqkEStBU8Vi (you) [..]
 └─ * patch id 191a14e520f2eeff7c0e3ee0a5523c5217eecb89
+└─ * revision id 191a14e520f2eeff7c0e3ee0a5523c5217eecb89
```

Comment:
Keep in mind the first revision's id is equal to the patch's id. So I wouldn't show it here unless they are different.

Reply:
Where do we want to place the revision in the 'patch list' output?

Here's the output as of now with the patch id.

```
 Define power requirements 191a14e520f2eeff7c0e3ee0a5523c5217eecb89 R0 3e674d1 (flux-capacitor-power) ahead 1, behind 0

  ● opened by did:key:z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk (you) 3 months ago                               

```

We need somewhere to include the Revision ID, the revision index is irrelevant now. Here is a suggestion where we use shortened ID's to suppress noise.

```
 Define power requirements 191a14e Rev  0f2eeff 3e674d1 (flux-capacitor-power) ahead 1, behind 0

  ● opened by did:key:z6Mkt67GdsW7715MEfRuP4pSZxJRJh6kj6Y48WRqVv4N1tRk (you) 3 months ago      
```

Reply:
Hmm that's too confusing, I have a better idea but this should be done in a separate PR

Reply:
![image](https://user-images.githubusercontent.com/40774/227280672-ba9e1cf1-82fc-4840-a1b1-985c35e65ac4.png)

In #478